x-pack/plugins/infra/public/types.ts

```diff
@@ -77,6 +77,7 @@ export interface InfraClientStartExports {
 }
 
 export interface InfraClientSetupDeps {
+  [x: string]: any;
```

Comment:
Is this a leftover? Don't see in the code why it's needed

Reply:
Thanks for spotting that, probably appeared after pulling main, removed